packages/beacon-node/test/unit/chain/rewards/blockRewards.test.ts

```diff
@@ -151,7 +154,7 @@ describe("chain / rewards / blockRewards", () => {
 
     const calculatedBlockReward = await computeBlockRewards(
       block.message,
-      preState as CachedBeaconStateAllForks,
+      preState as unknown as CachedBeaconStateAllForks,
```

Comment:
is this something we can improve cc @nflaig @nazarhussain @wemeetagain

Reply:
> is this something we can improve cc @nflaig @nazarhussain @wemeetagain

This is already solved. Fixed by removing `private` modifier in `EpochCache. historialValidatorLengths` See https://github.com/ChainSafe/lodestar/pull/6646#discussion_r1560632244